xarray/tests/test_computation.py

```diff
@@ -1554,6 +1553,35 @@ def test_covcorr_consistency(da_a, da_b, dim) -> None:
     assert_allclose(actual, expected)
 
 
+@requires_dask
+@pytest.mark.parametrize(
+    "da_a, da_b",
+    [
+        arrays_w_tuples()[1][i]
+        for i in range(len(arrays_w_tuples()[1]))
+        if i not in [5, 6]
+    ],  # TODO: arrays 5 and 6 make errors, why ?
+)
```

Comment:
```suggestion
@pytest.mark.parametrize("da_a, da_b", arrays_w_tuples()[1])
```
It now works with all arrays